float32 current_angle
float32 current_angular_velocity
float32 current_angular_acceleration
float32 desired_angle
float32 desired_angular_velocity
float32 desired_angular_acceleration
---
float32 flex_myobrick_pwm
float32 extend_myobrick_pwm